frontends/web/src/locales/cs/app.json

```diff
@@ -483,23 +480,23 @@
         },
         "privacyPolicy": "Zásady ochrany osobních údajů MoonPay",
         "protection": {
-          "description": "BitBoxApp při nákupu {{name}} neshromažďuje žádné údaje, s příchozími prostředky se zachází jako s běžnou transakcí. MoonPay potřebuje ke své činnosti shromažďovat některé osobní údaje. Jejich Zásady ochrany osobních údajů podrobně vysvětlují, jak je s těmito údaji nakládáno.",
-          "descriptionGeneric": "BitBoxApp při nákupu {{name}} neshromažďuje žádné údaje, s příchozími prostředky se zachází jako s běžnou transakcí. Partnerské burzy však ke svému fungování potřebují shromažďovat určité informace. Podrobnější informace o tom, jak je s daty nakládáno, naleznete v jejich zásadách ochrany osobních údajů.",
+          "descriptionGeneric_bitcoin": "BitBoxApp při nákupu kryptoměn neshromažďuje žádné údaje, s příchozími prostředky se zachází jako s běžnou transakcí. Partnerské burzy však ke svému fungování potřebují shromažďovat určité informace. Podrobnější informace o tom, jak je s těmito údaji nakládáno, naleznete v jejich zásadách ochrany osobních údajů.",
```

Comment:
@benma sorry there is still crypto in the text. this should be fixed in locize, if you have time ideally please pull one more time... 
if it is too late I think it is fine as is for this release.